stable/redis-ha/templates/redis-ha-service.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "redis-ha.fullname" . }}
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+  annotations:
+  {{- if .Values.serviceAnnotations }}
+{{ toYaml .Values.serviceAnnotations | indent 4 }}
+  {{- end }}
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+  - name: server
+    port: {{ .Values.redis.port }}
+    protocol: TCP
+    targetPort: redis
```

Comment:
Maybe `redis` should not be exposed as a service.

Application can not access redis master through this service, because it may access either master or slave.

Reply:
this is a headless service, a client (inside the cluster) can access the different pods conforming the statefulset by doing something like `redis-cli -h <redis-ha.fullname>-server-0.<redis-ha.fullname>` (or in my tests `redis-cli -h test-redis-ha-server-0.test-redis-ha`). 

If there is any need from external consumers to access this setup is to create a service which exposes (through ClusterIP or a LB) the current master. Of course, since this setup can elect a new master with each failure you'll need something in the middle to achieve that; we have a HAProxy based Helm Chart that takes care of this, discovering the elected master by performing checks on the available redis-ha stateless service instances. I can share this chart if proves to be helpful for this new redis-ha version.

Reply:
I see, thx.